### PR TITLE
Remove space from temporary file names

### PR DESCRIPTION
**Pull request by jan-cerny**
We use QTemporaryFile to create temporary files. According to
https://doc.qt.io/qt-5/qtemporaryfile.html, the default filename
is determined from QCoreApplication::applicationName(). It returns
"SCAP Workbench" which leads to creating temporary files with a space
character in their name, eg. "/tmp/SCAP Workbench.XM8663". The space
can cause problems with missing quotes s.a. the problems described in
https://github.com/OpenSCAP/scap-workbench/issues/196.
To avoid these problems, it's probably better to create the temporary
files without the space. I think I can use function setFileTemplate
for this purpose.

### CODE REVIEW

**Comment by jan-cerny** [comment]
Replaced by https://github.com/OpenSCAP/scap-workbench/pull/240

**Comment by matejak** [comment]
This is not a bad idea, although I would go for e.g. `SpacelessQTemporaryFile` declared in `Utils.h` that would extend the `QTemporaryFile` just by calling the `setFileTemplate` method. Then, if the code uses this class instead of the stock one, there won't be any space.

**Comment by jan-cerny** [comment]
I have changed it using @matejak's suggestion. I have noticed that we use our own version of class for creating temporary directories. That was needed when we used Qt 4, because Qt 4 didn't have QTemporaryDir class. But since SCAP Workbench release 1.2.0 we support only Qt 5, where this class is present, therefore we can drop the custom implementation and just use the QTemporaryDir from Qt 5 library.

File: src/Utils.cpp [hunk 1]
```diff
@@ -217,3 +217,9 @@ const QString& getSetSidPath()
     return ret;
 #endif
 }
+
+SpacelessQTemporaryFile::SpacelessQTemporaryFile () :
+    QTemporaryFile(QDir::cleanPath(QDir::tempPath() + "/" + "XXXXXX")) {}
+
+SpacelessQTemporaryDir::SpacelessQTemporaryDir () :
+    QTemporaryDir(QDir::cleanPath(QDir::tempPath() + "/" + "XXXXXX")) {}
```
Comment (matejak) [inline src/Utils.cpp]: Could you please add an indication that this is scap-workbench temp file/dir?

**Comment by jan-cerny** [comment]
@matejak I have added scap-workbench to temporary file name template.

**Comment by matejak** [comment]
Great, thanks!

**Pull request merged**